Userland/Applications/PixelPaint/Tools/PickerTool.cpp

```diff
@@ -22,6 +22,21 @@ void PickerTool::on_mousedown(Layer* layer, MouseEvent& event)
     m_editor->set_editor_color_to_color_at_mouse_position(layer_event, m_sample_all_layers);
 }
 
+void PickerTool::on_mouseup(Layer*, MouseEvent&)
+{
+    m_editor->set_appended_status_info(DeprecatedString::empty());
+}
+
+void PickerTool::on_mousemove(Layer* layer, MouseEvent& event)
+{
+    if (!layer)
+        return;
+    auto layer_event = event.layer_event();
+    if (!(layer_event.buttons() & GUI::MouseButton::Primary || layer_event.buttons() & GUI::MouseButton::Secondary))
+        return;
+    m_editor->set_editor_color_to_color_at_mouse_position(layer_event, m_sample_all_layers);
+}
```

Comment:
This change, again, is unrelated. You're allowing color picker to continously pick a new color while the mouse is being pressed, but that is not related to the status bar changes.

Reply:
Updated to two separate commits.